pandas/core/window/rolling.py

```diff
@@ -562,7 +563,7 @@ class BaseWindowGroupby(BaseWindow):
 
     def __init__(
```

Comment:
I don't know enough about `BaseWindowGroupby` but I think it could probably be a generic class `class BaseWindowGroupby(BaseWindow, Generic[FrameOrSeries])`?